corehq/apps/change_feed/document_types.py

```diff
@@ -67,18 +67,18 @@ def _get_document_type(document_or_none):
 
 def _case_doc_type_constructor(raw_doc_type, document):
     return DocumentMetadata(
-        raw_doc_type, CASE, document.get('type', None), _get_domain(document), _is_deletion(raw_doc_type)
+        raw_doc_type, CASE, document.get('type', None), _get_domain(document), is_deletion(raw_doc_type)
     )
 
 
 def _form_doc_type_constructor(raw_doc_type, document):
     return DocumentMetadata(
-        raw_doc_type, FORM, document.get('xmlns', None), _get_domain(document), _is_deletion(raw_doc_type)
+        raw_doc_type, FORM, document.get('xmlns', None), _get_domain(document), is_deletion(raw_doc_type)
     )
 
 
 def _domain_doc_type_constructor(raw_doc_type, document):
-    is_deletion = raw_doc_type == 'Domain-DUPLICATE' or _is_deletion(raw_doc_type)
+    is_deletion = raw_doc_type == 'Domain-DUPLICATE' or is_deletion(raw_doc_type)
```

Comment:
F823 local variable 'is_deletion' (defined in enclosing scope on line 117) referenced before assignment